client/scss/styleguide/_styleguide.scss

```diff
@@ -25,30 +22,72 @@
   }
 }
 
-.styleguide__hex {
+.styleguide__palette__heading {
+  @include font('HNM4');
+  text-transform: uppercase;
+  flex-basis: 100%;
+  margin-bottom: 1em;
+}
+
+.styleguide__palette__section {
+  display: flex;
+  flex-wrap: wrap;
+  border-bottom: 3px solid color('cream');
+  padding-bottom: 15px;
+  margin-bottom: 30px;
+
+  &:last-child {
+    border: 0;
+  }
+}
+
+.styleguide__palette__block {
+  flex-basis: 25%;
+  margin-bottom: 15px;
+}
+
+.styleguide__palette__color {
   position: relative;
-  width: 200px;
-  height: 200px;
+  min-width: 200px;
+  margin-right: 15px;
+  border: 1px solid color('transparent');
+
+  &--white {
+    border-color: color('accessible-grey-2');
+  }
+
+  &:before {
+    content: '';
+    display: block;
+    padding-top: 100%;
+  }
 }
 
-.styleguide__color {
-  position: absolute;
-  top: 5px;
-  left: 5px;
-  background: color('white');
-  padding: 5px;
-  max-width: 190px;
-  border-radius: 3px;
-  border: 1px solid color('smoke');
+.styleguide__palette__hyphen {
+  visibility: hidden;
+}
+
+.styleguide__palette__name {
```

Comment:
It still feels like metadata about the object that should not be incapsulated into it's style / classname. This is a coupling, which in itself sets off warning signs. I image if we had two `.palette__name`s, one in the site, and one in the styleguide, you'd probably set the base, and set the differences in a `palette__name--styleguide` modifier.

In short I wonder about storing the metadata that this element only exists in the styleguide in the classname (and superfluous metadata as it's not used for anything).

We could create a separate `styleguide.scss` file that is only included on the pattern pages which is a more explanitory I would imagine (and get's rid of the ambiguity), but also overkill in this case?

But true, ambiguity is bad.